src/pyodide/internal/python.ts

```diff
@@ -62,6 +64,11 @@ async function prepareWasmLinearMemory(Module: Module): Promise<void> {
   mountSitePackages(Module, SITE_PACKAGES.rootInfo);
   entropyMountFiles(Module);
   Module.noInitialRun = !SHOULD_RESTORE_SNAPSHOT;
+
+  // NB. loadPackages adds the packages to the `SITE_PACKAGES` global which then gets used in
+  // preloadDynamicLibs.
+  await loadPackages(Module, TRANSITIVE_REQUIREMENTS);
```

Comment:
I wonder line 70 and lines 64 and 65 should be moved into `loadPyodide`. Since they don't strictly speaking have anything to do with `prepareWasmLinearMemory`. And is it worth giving `loadPackages` its own jaeger span?